backend/app/api/endpoints/knowledge.py

```
@@ -767,3 +789,173 @@ def get_qa_history(
+
+    return SummaryRefreshResponse(
+        message="Summary refresh started",
+        status="generating",
+    )
+
+
+@summary_router.get("/{kb_id}/documents/{doc_id}/summary")
+async def get_document_summary(
+    kb_id: int,
+    doc_id: int,
+    current_user: User = Depends(security.get_current_user),
```

Ruff (0.14.10)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)